src/ansys/dpf/post/simulation.py

```diff
@@ -649,7 +668,9 @@ def _build_selection(
                 )
         if named_selections:
             selection.select_named_selection(
-                named_selection=named_selections, location=location
+                named_selection=named_selections,
+                location=location,
+                inclusive=requires_manual_averaging,
```

Comment:
Inclusive in case of "requires manual averaging" to be consistent with a selection by a node_ids array. Not inclusive for the standard case to be backward compatible.